locales/locales.json

```diff
@@ -307,7 +307,8 @@
       "offers": "Angebote",
       "events": "Veranstaltungen",
       "news": "Nachrichten",
-      "newsAlternative": "Neuigkeiten"
+      "newsAlternative": "Neuigkeiten",
+      "pois": "Points of Interest"
```

Comment:
These need to be added to the https://github.com/Integreat/integreat-locales/ project. After that you can pull in the locales from there.

Reply:
ok, can i just revert my changes for locales.json? @maxammann

Reply:
yes just revert :)